storage/remote/ewma.go

```diff
@@ -44,7 +46,9 @@ func newEWMARate(alpha float64, interval time.Duration) *ewmaRate {
 func (r *ewmaRate) rate() float64 {
 	r.mutex.Lock()
 	defer r.mutex.Unlock()
-	return r.lastRate
+	r.t.Add(1)
+	// for bias correction, lastRate = lastRate/(1-(1-alpha))^t
```

Comment:
For future readers it is probably worth adding a link to a reference as to what this is. Also, I believe this should be `lastRate = lastRate / (1 - (1 - alpha)^t)`. Your implementation looks correct to me, but not the comment.

Reply:
Hey everyone